docs/gen-ai/gen-ai-metrics.md

```diff
@@ -2,28 +2,32 @@
 linkTitle: Generative AI metrics
 --->
 
-# Semantic Conventions for Generative AI Client Metrics
+# Semantic Conventions for Generative AI Metrics
 
 **Status**: [Experimental][DocumentStatus]
 
-The conventions described in this section are specific to Generative AI client
-applications.
-
-**Disclaimer:** These are initial Generative AI client metric instruments
-and attributes but more may be added in the future.
-
 <!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
 
 <!-- toc -->
 
 - [Generative AI Client Metrics](#generative-ai-client-metrics)
   - [Metric: `gen_ai.client.token.usage`](#metric-gen_aiclienttokenusage)
   - [Metric: `gen_ai.client.operation.duration`](#metric-gen_aiclientoperationduration)
+- [Generative AI Model Server Metrics](#generative-ai-model-server-metrics)
+  - [Metric: `gen_ai.server.request_duration`](#metric-gen_aiserverrequest_duration)
+  - [Metric: `gen_ai.server.time_per_output_token`](#metric-gen_aiservertime_per_output_token)
+  - [Metric: `gen_ai.server.time_to_first_token`](#metric-gen_aiservertime_to_first_token)
```

Comment:
Can you please take a look at https://github.com/vllm-project/vllm/blob/main/vllm/tracing.py#L88-L94 , this is for server observability with vLLM, it has request id, best_of, time_in_queue etc, do you want to add those metrics as well?

Reply:
This was introduced by https://github.com/vllm-project/vllm/pull/4687/

Reply:
Yes, we can add those as well. But I'd do it in a separate change. Do you want to raise a PR for the tracing attributes?

Reply:
Yes, I can do that, thanks @achandrasekar